.github/workflows/my-ci-pipeline.yml

```
@@ -0,0 +1,18 @@
+name: Continuous Integration
+on: 
+  push:
+    paths-ignore:
+      - 'README.md'
+  pull_request:
+    paths-ignore:
+      - 'README.md'
+
+jobs:
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - run: docker build --target test --tag my-test-image .
+    - run: docker run --env-file .env.test my-test-image todo_app/tests
+    - run: docker run -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_API_TOKEN=${{ secrets.TRELLO_API_TOKEN }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} my-test-image todo_app/end_to_end_tests
```

Minor, but adding a "name" field to these steps might make your pipeline a little more readable when looking at them through the GitHub UI, e.g.
```suggestion
    - run: docker run -e TRELLO_API_KEY=${{ secrets.TRELLO_API_KEY }} -e TRELLO_API_TOKEN=${{ secrets.TRELLO_API_TOKEN }} -e SECRET_KEY=${{ secrets.SECRET_KEY }} my-test-image todo_app/end_to_end_tests
      name: Run the end-to-end tests
```